site/docs/foundations/color.mdx

```diff
@@ -2,10 +2,39 @@
 title: Color
 layout: DetailTechnical
 data:
-  status: In progress
+
 ---
 
-Colors can be used to communicate meaning, suggest a visual hierarchy and help to distinguish our products from others while providing a cohesive accessible experience.
+Colors can be used to communicate meaning, establish visual hierarchy, and differentiate Salt products from others, all while ensuring a cohesive and accessible experience. Colors are divided into several categories to communicate several concepts across the Salt design system.
```

Comment:
```suggestion
Colors can be used to communicate meaning, establish visual hierarchy, and differentiate products, all while ensuring a cohesive and accessible experience. Colors are divided into several categories to communicate several concepts across the Salt design system.
```

Do we have to limit this to Salt products? Also "from others" feels a bit redundant?